lib/RateLimiterRedis.js

```diff
@@ -37,6 +40,28 @@ class RateLimiterRedis extends RateLimiterStoreAbstract {
     }
   }
 
+  /**
+   * Prevent actual redis call if redis connection is not ready and insuranceLimiter is present
```

Comment:
@dmozgovoi I think, you should remove `and insuranceLimiter is present` from the comment since there is no check for that. And it is good, some may want to rely on Redis client status even when `insuranceLimiter` is not set.

Reply:
Done